### PR TITLE
Ddf 1211

### PR DESCRIPTION
**Pull request by ricklarsen**
@stustison @smithjosh

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/58)

### CODE REVIEW

File: distribution/docs/src/main/resources/ManContents.adoc [hunk 1]
```diff
@@ -1589,10 +1593,12 @@ After the keystore is successfully created, delete the jetty files used to perfo
...
-* Add system console to whitelisted contexts
-** `https://localhost:8993/system/console`
-*** Navigate to the Web Context Policy Manager
-*** Add /system/console to the Whitelisted Contexts
+* Add system console to whitelisted contexts.
+** `https://localhost:8993/admin`
+*** Select {branding} Security.
+*** Select *Configuration* tab.
+*** Select the *Web Context Policy Manager*.
+*** Add `/system/consoleKEEP` to the Whitelisted Contexts
```
Comment (stustison) [inline distribution/docs/src/main/resources/ManContents.adoc]: I think you have a typo here.

File: distribution/docs/src/main/resources/ManContents.adoc [hunk 2]
```diff
@@ -1731,13 +1737,15 @@ org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.
...
-* Remove system/console from whitelist
-** `https://localhost:8993/system/console`
-*** Navigate to the Web Context Policy Manager
-*** Remove `/system/console` from the Whitelisted Contexts
+* Remove system/consoleKEEP from whitelist
+** `https://localhost:8993/admin`
+*** Select {branding} Security.
+*** Select *Configuration* tab.
+*** Select the *Web Context Policy Manager*.
+*** Remove `/system/consoleKEEP` from the Whitelisted Contexts
```
Comment (stustison) [inline distribution/docs/src/main/resources/ManContents.adoc]: Typo?

File: distribution/docs/src/main/resources/ManContents.adoc [hunk 3]
```diff
@@ -1793,7 +1801,7 @@ An essential part to the {branding} Clustering solution is the ability to manage
...
+Before starting your {branding} cluster nodes, you must first setup your node network configurations. See the Configuring {branding} Clustering For Multicast & Unicast section for more information. To view all of the available nodes, navigate in your browser to the {branding} Admin Console (https://localhost:8993/admin) and click on the tab labeled “Clustered Groups”. Under the group “default”, you should see all running {branding} nodes that have been clustered. It is also possible to move all instances into a named group. In order to perform this action, you must have access to one of the {branding} command line shells directly or utilize Gogo located at https://localhost:8993/system/consoleKEEP/gogo.
```
Comment (stustison) [inline distribution/docs/src/main/resources/ManContents.adoc]: Typo?

File: distribution/docs/src/main/resources/ManContents.adoc [hunk 4]
```diff
@@ -1943,7 +1952,7 @@ As you can see, the multicast option has been set to true and the tcp-ip option
...
+In most cases, the {branding} system console should provide you with a listing of all features, repositories, and configurations that are installed on the cluster. There are times when the cluster can become out of sync. This instance may occur if a system has been offline for some time. One way to verify the synchronized lists of the cluster is to run cluster commands from the command line. In order to perform these actions, you must have access to one of the {branding} command line shells directly or through the use of Gogo (http://localhost:8993/system/consoleKEEP/gogo). Once at the command line, execute the following command to see the list of deployed features for your cluster:
```
Comment (stustison) [inline distribution/docs/src/main/resources/ManContents.adoc]: Typo?

File: distribution/docs/src/main/resources/ManContents.adoc [hunk 5]
```diff
@@ -2309,13 +2319,14 @@ org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.
...
 ** Search UI: https://ddf:8993/search
 
-* Remove system/console from whitelist
-** `https://localhost:8993/system/console`
+* Remove system/consoleKEEP from whitelist
+** `https://localhost:8993/admin`
+*** Navigate to DDF Security
 *** Navigate to the Web Context Policy Manager
-*** Remove `/system/console` from the Whitelisted Contexts
+*** Remove `/system/consoleKEEP` from the Whitelisted Contexts
```
Comment (stustison) [inline distribution/docs/src/main/resources/ManContents.adoc]: Typo.

File: distribution/docs/src/main/resources/ManContents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/ManContents.adoc]: fixed.

File: distribution/docs/src/main/resources/ManContents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/ManContents.adoc]: fixed.

File: distribution/docs/src/main/resources/ManContents.adoc (hunk 5 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/ManContents.adoc]: fixed.

**Pull request merged**